packages/xo-web/src/common/report-bug-button.js

```diff
@@ -18,38 +31,109 @@ export const reportBug = ({ formatMessage, message, title }) => {
   )
 
   window.open(
-    process.env.XOA_PLAN < 5
-      ? `https://xen-orchestra.com/#!/member/support?title=${encodedTitle}&message=${encodedMessage}`
-      : `https://github.com/vatesfr/xen-orchestra/issues/new?title=${encodedTitle}&body=${encodedMessage}`
+    `https://github.com/vatesfr/xen-orchestra/issues/new?title=${encodedTitle}&body=${encodedMessage}`
   )
 }
 
-const ReportBugButton = ({
-  formatMessage,
+const reportOnSupportPanel = async ({
+  files = [],
+  formatMessage = identity,
   message,
-  rowButton,
   title,
-  ...props
 }) => {
-  const Button = rowButton ? ActionRowButton : ActionButton
-  return (
-    <Button
-      {...props}
-      data-formatMessage={formatMessage}
-      data-message={message}
-      data-title={title}
-      handler={reportBug}
-      icon='bug'
-      tooltip={_('reportBug')}
-    />
-  )
+  const { FormData, open } = window
+
+  const formData = new FormData()
+  if (title !== undefined) {
+    formData.append('title', title)
+  }
+  if (message !== undefined) {
+    formData.append('message', formatMessage(message))
+  }
+  files.forEach(({ content, name }) => {
+    formData.append('attachments', content, name)
+  })
+
+  await Promise.all([
+    xoaUpdater
+      .getLocalManifest()
+      .then(
+        manifest =>
+          formData.append(
+            'attachments',
+            createBinaryFile(JSON.stringify(manifest, null, 2)),
+            'manifest.json'
+          ),
+        error => logger.warn('cannot get the local manifest', { error })
+      ),
+    checkXoa().then(
```

Comment:
This can be long, or even stuck, how do we handle this?

Reply:
Normally the plugin can handle this case to all time returns a value.